tests/main/views/test_briefs.py

```diff
@@ -744,6 +744,34 @@ def test_withdraw_a_brief(self):
         assert data['briefs']['status'] == 'withdrawn'
         assert data['briefs']['withdrawnAt'] is not None
 
+    def test_cancel_a_brief(self):
+        self.setup_dummy_briefs(1, title='The Title', status='closed')
+
+        res = self.client.post(
+            '/briefs/1/cancel',
+            data=json.dumps({
+                'update_details': {'updated_by': 'example'}
+            }),
+            content_type='application/json')
+        data = json.loads(res.get_data(as_text=True))
+
+        assert res.status_code == 200
+        assert data['briefs']['status'] == 'cancelled'
+
+    def test_update_a_brief_as_unsuccessful(self):
+        self.setup_dummy_briefs(1, title='The Title', status='closed')
+
+        res = self.client.post(
+            '/briefs/1/unsuccessful',
+            data=json.dumps({
```

Comment:
Just had to test this but we don't actually have to `json.dumps` the data. Although given it is consistent with the rest of the file it's the right thing to do here.